mosip-identity-certify-plugin/pom.xml

```diff
@@ -153,6 +153,17 @@
                </exclusion>
            </exclusions>
         </dependency>
+        <dependency>
+            <groupId>io.mosip.esignet</groupId>
+            <artifactId>esignet-integration-api</artifactId>
+            <version>1.3.1-SNAPSHOT</version>
```

Comment:
since we used wild card * in esignet-core dependency, all the transitive dependencies are excluded including esignet-integration-api. this dependency is needed for OIDCTransaction class

Reply:
then we don't need the esignet-core right ?

Reply:
esignet-core has OIDCTransaction class and esignet-integration-api has dto's that are imported in to OIDCTransaction class